python/lsst/drp/tasks/assemble_coadd.py

```diff
@@ -392,6 +392,9 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
             skyMap, tractId=outputDataId["tract"], patchId=outputDataId["patch"]
         )
 
+        # Attach the bounding box of the coadd this task is producing.
+        self.coadd_bbox = inputData["skyInfo"].bbox
```

Comment:
I'm not a fan of putting per-quantum state on the task instance.  Better to pass this around as another argument if it's needed in multiple methods.